libs/providers/flagsmith/README.md

```diff
@@ -0,0 +1,93 @@
+# Flagsmith Provider
+
+This provider is an implementation for the [JavaScript SDK](https://docs.flagsmith.com/clients/javascript/) of [Flagsmith](https://flagsmith.com).
+
+## Installation
+
+```
+npm install @openfeature/flagsmith-provider @openfeature/web-sdk
```

Comment:
NPM will auotmatically install peers, and keep them up to data according to compatibility. Since you've added the web-sdk as a peer (or maybe our tooling did) I would recommend you remove the manual installation for NPM.

Reply:
Thanks, made me realise I should have added a flagsmith peer dep too.